tests/test_main.py

```diff
@@ -918,6 +918,10 @@ class MyModel(BaseModel):
 
     assert MyModel.parse_obj(['a']).__root__ == ['a']
     assert MyModel.parse_obj({'__root__': ['a']}).__root__ == ['a']
+    with pytest.raises(ValidationError, match='...'):
+        MyModel.parse_obj({'__not_root__': ['a']})
+    with pytest.raises(ValidationError, match='...'):
```

Comment:
We need something in the match! :smile: (I was being lazy).

Reply:
Haha okay hold on

Reply:
Should be better now